client_plugin/drivers/shared/etcdops.go

```diff
@@ -72,10 +77,32 @@ type kvPair struct {
 	value string
 }
 
+type kvStore interface {
```

Comment:
Is it intended that the interface is not exported, but the functions in it are exported?